src/components/shared/AvatarUploader/AvatarUploader.tsx

```diff
@@ -100,51 +110,69 @@ const AvatarUploader = ({
     dropzone: styles.dropzoneNoButtonsVariant,
   };
 
-  // Formik is used for state and error handling through FileUpload, nothing else
   return (
-    <Formik onSubmit={() => {}} initialValues={{ avatarUploader: [] }}>
-      <form>
-        <FileUpload
-          elementOnly={elementOnly}
-          classNames={hasButtons ? styles : noButtonStyles}
-          dropzoneOptions={{
-            accept: ACCEPTED_MIME_TYPES,
-            maxSize: ACCEPTED_MAX_FILE_SIZE,
-            disabled,
-          }}
-          label={label}
-          labelAppearance={labelAppearance}
-          help={help}
-          maxFilesLimit={1}
-          name="avatarUploader"
-          renderPlaceholder={placeholder}
-          ref={dropzoneRef}
-          itemComponent={AvatarUploadItem}
-          upload={upload}
-          handleError={handleError}
-          dataTest="avatarUploaderDrop"
-        >
-          {renderOverlay()}
-        </FileUpload>
-        {hasButtons && (
-          <div className={styles.buttonContainer}>
-            <Button
-              appearance={{ theme: 'danger' }}
-              text={{ id: 'button.remove' }}
-              onClick={remove}
-              disabled={!isSet}
-              dataTest="avatarUploaderRemove"
-            />
-            <Button
-              text={{ id: 'button.choose' }}
-              onClick={choose}
-              dataTest="avatarUploaderChoose"
-            />
-          </div>
-        )}
-      </form>
-    </Formik>
+    <Form onSubmit={() => {}} defaultValues={{ avatarUploader: [] }}>
+      {({ formState: { errors }, reset }) => {
+        const name = 'avatarUploader';
+        const error = errors[name];
+        const hasError = !!uploadError || !!error;
+        const choose = () => {
+          reset();
+          // waits for form state to reset
+          setTimeout(open, 0);
```

Comment:
I think this is our best option given the current design of the file uploader. If you upload an incorrect file, it will disable the form because the limit on the `AvatarUploader` is 1, and the form state is storing the one, incorrect file. 

To re-enable the form, we hit `reset()`, which clears form state and thus sets the file numbers back below the limit (to 0, specifically). 

The timeout ensures that we open the file explorer after the form state has been cleared (and thus the dropzone is no longer disabled).